contents/docs/cdp/destinations/tiktok-ads.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Send PostHog conversion events to TikTok Ads
+templateId: template-tiktok-ads
+---
+
+import Requirements from "../_snippets/requirements.mdx"
+import FeedbackQuestions from "../_snippets/feedback-questions.mdx"
+import PostHogMaintained from "../_snippets/posthog-maintained.mdx"
+
+<Requirements />
+
+You'll also need access to the relevant TikTok Ads account.
+
+## Installation
+
+1. In PostHog, click the [Data pipeline](https://us.posthog.com/pipeline/overview) tab in the left sidebar.
+
+2. Click the **Destinations** tab.
+
+3. Search for **TikTok Ads Conversions** and select the destination.
```

Comment:
Doesn't seem to be available yet?
![CleanShot 2024-11-25 at 11 28 53@2x](https://github.com/user-attachments/assets/187eaf14-16a5-491a-867d-ae1800af135b)